Source/Core/Core/HW/GCMemcard/GCMemcardDirectory.cpp

```diff
@@ -123,7 +147,8 @@ std::vector<std::string> GCMemcardDirectory::GetFileNamesForGameID(const std::st
     if (!gci_file.ReadBytes(&gci.m_gci_header, Memcard::DENTRY_SIZE))
       continue;
 
-    const std::string gci_filename = gci.m_gci_header.GCI_FileName();
+    const std::string gci_filename =
```

Comment:
Not really relevant to the logic at hand but this could probably _not_ be `const` and instead then be moveable into the `loaded_saves` on line 172.

Reply:
nm, I see the next commit :)